bisulfite/pileup.cwl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Biscuit pileup"
+baseCommand: ["/usr/bin/biscuit", "pileup"]
+stdout: pileup.vcf.gz
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 16000
+      coresMin: 4
+arguments: [
+    { valueFrom: "-q", position: -10 },
+    { valueFrom: $(runtime.cores), position: -9 },
+    { valueFrom: "-w", position: -8 },
+    { valueFrom: "pileup_stats.txt", position: -7 },
+    { shellQuote: false, valueFrom: "|" },
+    "/bin/bgzip"
```

Comment:
In the current version of `mgibio/bisulfite`, `bgzip` is located at `/opt/htslib/bin/bgzip`.

Reply:
yeah, just found that during my test run - push coming shortly